images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -37,6 +37,18 @@ RUN set -ex \
         swig \
         tk-dev \
         uuid-dev \
+        python3-dev \
```

Comment:
Adding additional libraries to support all optional python modules.  
![image](https://user-images.githubusercontent.com/86327553/217119318-1442573c-9ce0-454d-8d89-37cc9c898fb2.png)
https://devguide.python.org/getting-started/setup-building/index.html#install-dependencies

Reply:
Would you mind adding a comment with this link above the lines where we're installing the Python 3 dependencies?